### ISSUE TITLE
Just tried to upgrade to 8.11.2-1 from 8.10.6

### ISSUE THREAD

**Issue by pnelsonsr** [sameersbn/docker-gitlab#875]
Tried to upgrade (we do them once a week) following the process (which has worked find up to this upgrade);

backup gitlab
stop gitlab
remove gitlab
pull new gitlab
start gitlab

But this time when it gets to:

Configuring gitlab::database

after a few moments is drops to the command line.  Tried this on several systems and it fails on every one.  I checked pgsql and it working fine.  I checked a 8.10.6 system and it works fine, then I upgrade and it stops wat gitlab::database like the others.  

Can you help me get this working with 8.11.2-1?  Where do I look?

**Comment by SolidNerd** [comment]
Hey, 
did you link your postgresql container to gitlab backup container ? Otherwise it will not work. For more information you could enable `DEBUG` with `DEBUG=true` in the gitlab container.

**Comment by pnelsonsr** [comment]
Using an external postgresql database that is...  Its not in a container.  This use to work but stop working when I tried this version.

**Comment by bdwilton** [comment]
I went from 8.9.6-1 to 8.11.2-1 but need to set some new environment vars, in upgrade notes it says...
Note: Since GitLab 8.11.0 you need to provide the GITLAB_SECRETS_SECRET_KEY_BASE and GITLAB_SECRETS_OTP_KEY_BASE parameters while starting the image. These should initially both have the same value as the contents of the /home/git/data/.secret file.
Did you set those ?

**Comment by pnelsonsr** [comment]
No I do not have either of those.  I did have a GITLAB_SECRETS_DB_KEY_BASE but not either GITLAB_SECRETS_SECRET_KEY_BASE and GITLAB_SECRETS_OTP_KEY_BASE.  So I tried changing GITLAB_SECRETS_DB_KEY_BASE to GITLAB_SECRETS_SECRET_KEY_BASE and then copying the value to the added parameter GITLAB_SECRETS_OTP_KEY_BASE.  This did not work.  So I went looking for the .secret file and I could not find it.  I think this might be because GITLAB is not starting.  It stops during Configuring gitlab::database phase of startup so I can not log into the gitlab docker instance because it shows as exited.  I'm not sure what to do here.  But my gitlab instance has not be successfull at starting in any version past 8.10.6
